web/components/ConferenceUpdatesForm/ConferenceUpdatesForm.module.css

```diff
@@ -107,3 +116,10 @@
     margin: 0;
   }
 }
+
+@keyframes item-enter {
+  from {
+    opacity: 0;
+    transform: translateY(var(--distance)) rotate(var(--rotation));
+  }
+}
```

Comment:
Should this animation perhaps be pulled up into _globals.css_ rather than duplicated many places?